aries/aries_vcx/tests/utils/test_agent.rs

```diff
@@ -103,15 +103,14 @@ where
     W: BaseWallet,
 {
     let acme = create_test_agent(genesis_file_path.to_string()).await;
-    let acme_vk =
-        get_verkey_from_wallet(acme.wallet.get_wallet_handle(), &acme.institution_did).await?;
+    let acme_vk = acme.wallet.did_key(&acme.institution_did).await?;
```

Comment:
The original `get_verkey_from_wallet` seem to be dead code now, can be deleted. 

Along with that I've also noticed `libindy_replace_keys_start`, `libindy_replace_keys_apply` to be dead code now. 
We should review why clippy is not catching this, but let's look into that separately elsewhere. There might be possibly more dead code across the codebase we are unaware of.